sw/device/silicon_creator/mask_rom/boot_policy.c

```diff
@@ -29,9 +32,9 @@ rom_error_t boot_policy_manifest_check(const manifest_t *manifest) {
     return kErrorBootPolicyBadLength;
   }
   RETURN_IF_ERROR(manifest_check(manifest));
-  // TODO(#7879): Implement anti-rollback.
-  uint32_t min_security_version = 0;
-  if (manifest->security_version < min_security_version) {
+  boot_data_t boot_data;
+  RETURN_IF_ERROR(boot_data_read(lc_state, &boot_data));
+  if (manifest->security_version < boot_data.min_security_version_rom_ext) {
```

Comment:
This is a good target for future hardening. Can we add a comment here about that?